text/0000-javascript-module-api.md

```diff
@@ -0,0 +1,1058 @@
+| ---                                             | ---                      |
+| `import { run } from "@ember/runloop"`          | `Ember.run`              |
+| `import { begin } from "@ember/runloop"`        | `Ember.run.begin`        |
+| `import { bind } from "@ember/runloop"`         | `Ember.run.bind`         |
+| `import { cancel } from "@ember/runloop"`       | `Ember.run.cancel`       |
+| `import { debounce } from "@ember/runloop"`     | `Ember.run.debounce`     |
+| `import { end } from "@ember/runloop"`          | `Ember.run.end`          |
+| `import { join } from "@ember/runloop"`         | `Ember.run.join`         |
+| `import { later } from "@ember/runloop"`        | `Ember.run.later`        |
+| `import { next } from "@ember/runloop"`         | `Ember.run.next`         |
+| `import { once } from "@ember/runloop"`         | `Ember.run.once`         |
+| `import { schedule } from "@ember/runloop"`     | `Ember.run.schedule`     |
+| `import { scheduleOnce } from "@ember/runloop"` | `Ember.run.scheduleOnce` |
+| `import { throttle } from "@ember/runloop"`     | `Ember.run.throttle`     |
+
+### `@ember/service`
+| Module                                    | Global                 |
+| ---                                       | ---                    |
+| `import Service from "@ember/service"`    | `Ember.Service`        |
+| `import { inject } from "@ember/service"` | `Ember.inject.service` |
+
+### `@ember/string`
+| Module                                       | Global                    |
+| ---                                          | ---                       |
+| `import { camelize } from "@ember/string"`   | `Ember.String.camelize`   |
+| `import { capitalize } from "@ember/string"` | `Ember.String.capitalize` |
+| `import { classify } from "@ember/string"`   | `Ember.String.classify`   |
+| `import { dasherize } from "@ember/string"`  | `Ember.String.dasherize`  |
+| `import { decamelize } from "@ember/string"` | `Ember.String.decamelize` |
+| `import { fmt } from "@ember/string"`        | `Ember.String.fmt`        |
+| `import { htmlSafe } from "@ember/string"`   | `Ember.String.htmlSafe`   |
+| `import { loc } from "@ember/string"`        | `Ember.String.loc`        |
+| `import { underscore } from "@ember/string"` | `Ember.String.underscore` |
+| `import { w } from "@ember/string"`          | `Ember.String.w`          |
+
+### `@ember/utils`
+| Module                                     | Global            |
+| ---                                        | ---               |
+| `import { isBlank } from "@ember/utils"`   | `Ember.isBlank`   |
+| `import { isEmpty } from "@ember/utils"`   | `Ember.isEmpty`   |
+| `import { isNone } from "@ember/utils"`    | `Ember.isNone`    |
+| `import { isPresent } from "@ember/utils"` | `Ember.isPresent` |
+| `import { tryInvoke } from "@ember/utils"` | `Ember.tryInvoke` |
+| `import { typeOf } from "@ember/utils"`    | `Ember.typeOf`    |
+
```

Comment:
@fsanvido I think that `Ember.K` is a pretty low-level thing that probably shouldn't be too exposed. 
Besides, unless I stand corrected, I think that it was more or less a trick to avoid people writing more code and creating more function instances than necessary, but given how modules are transpiled I have the feeling (no empirical data) that that all the boilerplate that module transpilation adds would defeat any perf/size improvement of reusing the same `Ember.K` instance.